docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -138,6 +138,62 @@
                 }
 
   ###
+  # apply a patch update to a list of applications
+  #
+  patch:
+    description:
+      Change multiple existing applications by applying a patch.
+      All instances of these applications get replaced by the new version.
+      The order of dependencies will be applied correctly.
+      Each upgradeStrategy defines the behaviour of the upgrade for the related app.
+
+      The whole operation fails if the IDs of one or more applications are unknown.
+      The order of dependencies will be applied correctly.
+
+      If you have more complex scenarios with upgrades, use the groups endpoint.
+
+      Note&#58;  This operation will create a deployment. The operation finishes, if the deployment succeeds.
+       You can query the deployments endoint to see the status of the deployment.
+    is: [ secured, deployable ]
+    queryParameters:
+    body:
+      application/json:
+        example: !include examples/apps_create.json
+        type: app.App[]
```

Comment:
That's unfortunate – I haven't found a way to introduce an AppUpdate type in raml w/o breaking other things. A patch w/o id is allowed and works but the doc will state that an id is required.